message_definitions/v1.0/common.xml

```diff
@@ -1305,7 +1322,7 @@
       <entry value="207" name="MAV_CMD_DO_FENCE_ENABLE" hasLocation="false" isDestination="false">
         <description>Mission command to enable the geofence</description>
         <param index="1" label="Enable" minValue="0" maxValue="2" increment="1">enable? (0=disable, 1=enable, 2=disable_floor_only)</param>
-        <param index="2">Empty</param>
+        <param index="2" label="Types" enum="FENCE_TYPE">Fence types to enable or disable as a bitmask.  This parameter is ignored if param 1 has the value 2</param>
```

Comment:
need to call out that 0 means change all

Reply:
Fixed